qiling/os/uefi/uefi.py

```diff
@@ -16,6 +16,7 @@ def __init__(self, ql):
         super(QlOsUefi, self).__init__(ql)
         self.ql = ql
         self.entry_point = 0
+        self.pid = 0
```

Comment:
i think we got a OS wide pid. can u use that ?

Reply:
my bad it was for posix. i will move it to QlOS so u can use it

Reply:
Done moving, will be be able change it ?

Reply:
Done. I've added a small change in pe_uefi.